### PR TITLE
Query Browser: Add page Actions menu and kebab menu for each query

### PR DESCRIPTION
**Pull request by kyoto**

### CODE REVIEW

File: frontend/public/components/graphs/_graphs.scss [hunk 1]
```diff
@@ -78,8 +78,17 @@
 }
 
 .query-browser__table-toggle {
+  cursor: pointer;
```
Comment (spadgett) [inline frontend/public/components/graphs/_graphs.scss]: This is likely an accessibility problem we should fix.

File: frontend/public/components/graphs/_graphs.scss [hunk 2]
```diff
@@ -78,8 +78,17 @@
...
   font-size: 30px;
   font-weight: 600;
   padding-left: 4px;
   width: 38px;
 }
+
+.query-browser__query {
```
Comment (spadgett) [inline frontend/public/components/graphs/_graphs.scss]: nit: alpha

File: frontend/public/components/graphs/_graphs.scss (hunk 1 above)
Comment (kyoto) [reply frontend/public/components/graphs/_graphs.scss]: 👍 Good point, thanks

**Comment by kyoto** [comment]
Thanks for the suggestions @spadgett. I made some changes:
- Sorted the `.query-browser__*` CSS rules A->Z
- Made the collapse / expand toggle a `<button>`
- Also added a "Disable query" / "Enable query" kebab menu action

**Pull request merged**